mods/parsoid.js

```diff
@@ -784,7 +784,8 @@ PSP.makeTransform = function(from, to) {
 
     return function(restbase, req) {
         var rp = req.params;
-        if (!req.body || !req.body[from]) {
+        if ((!req.body && req.body !== '')
```

Comment:
Actually, will spec-based checking allow us to supply an empty body at all, @Pchelolo ?